src/app/app.component.ts

```diff
@@ -14,6 +14,15 @@ export class AppComponent {
     router.events.subscribe((url: any) => {
       this.intro = url.url !== "/" && url.url !== "/wizard";
     });
+
+    router.events.distinctUntilChanged((previous: any, current: any) => {
+      if (current instanceof NavigationEnd) {
+        return previous.url === current.url;
+      }
+      return true;
+    }).subscribe((x: any) => {
+      window['analytics'].page();
```

Comment:
I think we should add some useful information here, as explained in the docs for the `page()` method here: https://segment.com/docs/spec/page/

I guess it could be as simple as just passing the step nr, or the text "Step N", as the name of the page. That way we can distinguish between the pages that the user was on. (And we'd probably need a separate one for the intro page, or call it "Step 0" perhaps).

Does this also get called when people go back a page or when they directly click on the page numbers at the top?

Reply:
don't have that much information at this point so added the url it takes the from `/all/launchpad-new-project/1/` so we can see what step it was.

this function is called on every navigation regardless if back button of the browser was used or step numbers are clicked